src/v2/client/handleConfiguredFlow.js

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) 2021, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Errors from 'util/Errors';
```

Comment:
Please add some comments above each functions that devs should be aware of..

Reply:
comments added

Reply:
thanks..

On top of this file, can we also add what configured flows are, how, why and where it will be used. In very short and concise manner.. 

Probably we will have some kind of documentation for this in widget repo right ?

Reply:
Formal documentation is still to come. We want to release this more as a "beta" (undocumented) feature so a few customers can play with it and provide us with feedback before we release a more finalized version (with documentation)